api/v1alpha1/groupversion_info.go

```diff
@@ -29,10 +27,17 @@ import (
 )
 
 var (
+	// GroupVersion is group version used to register these objects
+	GroupVersion = schema.GroupVersion{Group: "submariner.io", Version: "v1alpha1"}
+
+	// SchemeBuilder is used to add go types to the GroupVersionKind scheme
+	SchemeBuilder = &scheme.Builder{GroupVersion: GroupVersion}
+
+	// AddToScheme adds the types in this group-version to the given scheme.
+	AddToScheme = SchemeBuilder.AddToScheme
+
+	//TODO (Jaanki) Remove below var once clientset usage is removed.
```

Comment:
I'm not clear on this comment - can you elaborate as to why?

Reply:
well, the new operator framework creates `GroupVersion` var instead of `SchemeGroupVersion`. `SchemeGroupVersion` is being used in `pkg/client/clientset` so I had to add it here. But since `clientset` code is now removed, this can also be removed.